app/Http/Controllers/Store/CentreController.php

```diff
@@ -60,17 +62,28 @@ public function exportRegistrationsSummary()
         // Get now()
         $now = Carbon::now();
 
-        // Get centres
-        $centres = $user->relevantCentres();
-
-        // get registrations
-        $registrations = Registration::whereIn(
-            'centre_id',
-            $centres->pluck('id')->all()
-        )
-            ->with(['centre','family.children','family.carers'])
+        // Get centre ids
+        $centre_ids = $user->relevantCentres()->pluck('id')->all();
```

Comment:
Do you need the `all()`. I can see it was in the `whereIn` before, but not sure what the diff would be without.

I think pluck returns an array of key-value pairs. Would have to look up.

Reply:
pluck returns a collection, with an property called all that has the results.